import "google/protobuf/any.proto";

option java_outer_classname = "MdBarDataBO_PRO";
option java_package = "com.purefun.fams.md.pro";

message MdBarDataBO {
    optional string uuid = 1;
    optional sint64 boid = 2;
    optional string destination = 3;
    optional string security_code = 4;
    optional string exch = 5;
    optional string security_type = 6;
    optional string date = 7;
    optional double open = 8;
    optional double high = 9;
    optional double low = 10;
    optional double close = 11;
    optional sint64 volume = 12;
    optional double change = 13;
    optional double pre_close = 14;
    optional double pct_chg = 15;
    optional double amount = 16;
}
